seam/exceptions.py

```diff
@@ -0,0 +1,43 @@
+from niquests import Response
+from .routes.models import ActionAttempt
+
+
+# HTTP
+class SeamApiException(Exception):
```

Comment:
I went back and forth on this name, but I landed on `SeamHttpApiError` in the JS SDK, and we should be consistent in the other projects. This gives us some space in the future to introduce errors from the Seam API that are not from the HTTP layer.

```suggestion
class SeamHttpApiError(Exception):
```

Reply:
Ah, there is already an issue for this :) https://github.com/seamapi/python-next/issues/48

Reply:
Yep and [PR](https://github.com/seamapi/python-next/pull/58) is also ready!